app/models/calendar_event.rb

```diff
@@ -85,8 +85,10 @@ def self.project_date_filter(range, scope = Project)
   end
 
   def self.project_step_date_filter(range, scope = ProjectStep)
-    scope.where("actual_end_date BETWEEN :first AND :last OR scheduled_start_date BETWEEN :first AND :last "\
-      "OR old_start_date BETWEEN :first and :last", {first: range.first, last: range.last})
+    scope.
+    where("actual_end_date BETWEEN :first AND :last OR scheduled_start_date BETWEEN :first AND :last "\
+    "OR old_start_date BETWEEN :first and :last", {first: range.first, last: range.last}).
+    where(is_finalized: true)
```

Comment:
@arisah-ijeoma Sometimes we will need to see the steps that are not finalized in the calendar. So this needs to be its own filter/method.

In this feature, only finalized steps show up in the general calendar. But steps not finalized would still show up in a loan's or project's calendar. Dashboard will show finalized only. (I'm going to add another issue for the dashboard specifically).

Thank you!